site/src/components/AuthAndFrame/AuthAndFrame.tsx

```diff
@@ -15,14 +17,37 @@ interface AuthAndFrameProps {
  * Wraps page in RequireAuth and renders it between Navbar and Footer
  */
 export const AuthAndFrame: FC<AuthAndFrameProps> = ({ children }) => {
-  const styles = useStyles()
+  const styles = useStyles({})
   const xServices = useContext(XServiceContext)
+  const [authState] = useActor(xServices.authXService)
   const [buildInfoState] = useActor(xServices.buildInfoXService)
+  const [updateCheckState, updateCheckSend] = useActor(
+    xServices.updateCheckXService,
+  )
+
+  useEffect(() => {
+    if (authState.matches("signedIn")) {
+      updateCheckSend("CHECK")
+    } else {
+      updateCheckSend("CLEAR")
+    }
+  }, [authState, updateCheckSend])
```

Comment:
review(site): I'm not super happy with how this turned out. I kind of wanted to represent this in the updateCheckXService, but I couldn't figure out how to do inter-machine dependencies.

This snippet here enables e.g. the following scenario:

1. User is loggedin as non-owner
2. User logs out
3. User logs in as owner
4. Update notification is shown

Reply:
Yeah, inter-machine dependencies are tricky; I'm not sure if there's a better way. @presleyp do you have any workarounds? I think you looked at something like this recently with pagination.